user_docs/en/changes.t2t

```diff
@@ -28,9 +28,17 @@ Please refer to [the developer guide https://www.nvaccess.org/files/nvda/documen
 
 - Note: this is an Add-on API compatibility breaking release.
 Add-ons will need to be re-tested and have their manifest updated.
-- Updated Comtypes to version 1.2.0. (#15513)
-- Updated fast_diff_match_patch to version 2.0.1. (#15514)
-- Updated SCons to version 4.5.2. (#15529)
+- NVDA is now built with Python 3.11 (#12064)

```

Comment:
Style consistency:

```suggestion
- Updated Python to 3.11 (#12064)
```

Reply:
These are separate items. 
Python is part of the build process, it's not an internal project dependency. We haven't updated a version of Python internally to the project, we've updated what we use to build the project.

Reply:
Makes sense. Both previous Python build-time updates use the "updated Python to x" language.